include/ringmesh/mesh/cartesian_grid.h

```diff
@@ -165,9 +165,10 @@ namespace RINGMesh
                 }
                 else
                 {
-                    Logger::warn( "Point ", coords,
-                        " has indexes outside of the cartesian grid limits." );
-                    return -1;
+                    //                    Logger::warn( "Point ", coords,
+                    //                        " has indexes outside of the
```

Comment:
Choose but do not leave commented code

Reply:
Ok, I'll leave it for now, and remove it later if it proves to be more a strain than a help.